huaweicloud/services/css/resource_huaweicloud_css_manual_log_backup.go

```diff
@@ -0,0 +1,291 @@
+	finishedAt := utils.PathSearch("finishedAt", resp, float64(0)).(float64) / 1000
+	mErr := multierror.Append(
+		nil,
+		d.Set("region", region),
+		d.Set("job_id", utils.PathSearch("jobId", resp, nil)),
+		d.Set("type", utils.PathSearch("jobTypes", resp, nil)),
+		d.Set("status", utils.PathSearch("status", resp, nil)),
+		d.Set("log_path", utils.PathSearch("logPath", resp, nil)),
+		d.Set("created_at", utils.FormatTimeStampRFC3339(int64(createdAt), false)),
+		d.Set("finished_at", utils.FormatTimeStampRFC3339(int64(finishedAt), false)),
+		d.Set("failed_msg", utils.PathSearch("failedMsg", resp, nil)),
+	)
+	return diag.FromErr(mErr.ErrorOrNil())
+}
+
+func resourceManualLogBackupDelete(_ context.Context, _ *schema.ResourceData, _ interface{}) diag.Diagnostics {
+	errorMsg := "Deleting manual log backup resource is not supported. The manual log backup resource" +
+		"is only removed from the state, the manual log backup content and record remain in the cloud."
+	return diag.Diagnostics{
+		diag.Diagnostic{
+			Severity: diag.Warning,
+			Summary:  errorMsg,
+		},
+	}
+}
+
+func checkOpeningLogBackupFunc(client *golangsdk.ServiceClient, clusterID string) error {
+	getLogSettingHttpUrl := "v1.0/{project_id}/clusters/{cluster_id}/logs/settings"
+	getLogSettingPath := client.Endpoint + getLogSettingHttpUrl
+	getLogSettingPath = strings.ReplaceAll(getLogSettingPath, "{project_id}", client.ProjectID)
+	getLogSettingPath = strings.ReplaceAll(getLogSettingPath, "{cluster_id}", clusterID)
+
+	getLogSettingPathOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		MoreHeaders:      map[string]string{"Content-Type": "application/json"},
+	}
+
+	getLogSettingResp, err := client.Request("GET", getLogSettingPath, &getLogSettingPathOpt)
+	if err != nil {
+		return err
+	}
+
+	getLogSettingRespBody, err := utils.FlattenResponse(getLogSettingResp)
+	if err != nil {
+		return err
+	}
+
+	logSetting := utils.PathSearch("logConfiguration", getLogSettingRespBody, nil)
+	logSwitch := utils.PathSearch("logSwitch", logSetting, false).(bool)
+
+	if logSetting == nil || !logSwitch {
+		return fmt.Errorf("before backing up logs, please enable the log backup function")
+	}
+
+	return nil
+}
+
+func createLogBackup(client *golangsdk.ServiceClient, clusterID string) error {
+	createLogBackupHttpUrl := "v1.0/{project_id}/clusters/{cluster_id}/logs/collect"
+	createLogBackupPath := client.Endpoint + createLogBackupHttpUrl
+	createLogBackupPath = strings.ReplaceAll(createLogBackupPath, "{project_id}", client.ProjectID)
+	createLogBackupPath = strings.ReplaceAll(createLogBackupPath, "{cluster_id}", clusterID)
+
+	createLogBackupPathOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		MoreHeaders:      map[string]string{"Content-Type": "application/json"},
+	}
+
+	_, err := client.Request("POST", createLogBackupPath, &createLogBackupPathOpt)
+	if err != nil {
+		return fmt.Errorf("error manual backup log, err: %s", err)
```

Comment:
```suggestion
		return fmt.Errorf("error creating manual backup log, err: %s", err)
```
and fix other places